workflow/type.go

```diff
@@ -15,8 +25,43 @@ type task struct {
 	TaskKey      string
 }
 
+type filter struct {
+	Key       string
+	Predicate predicate
+	Value     interface{}
+}
+
 // Trigger is an event that triggers a workflow
 type trigger struct {
 	InstanceHash hash.Hash
 	EventKey     string
+	Filters      []*filter
+}
+
+func (t *trigger) Match(evt *event.Event) (bool, error) {
```

Comment:
```suggestion
func (t *trigger) Match(evt *event.Event) (bool) {
```
No error is returned.

Reply:
good catch, fixed